spec/mailers/project_mailer_spec.rb

```diff
@@ -20,8 +20,8 @@
   end
 
   describe '#comment' do
-    let!(:comment) { create :comment, project: project }
-    let!(:commenter) { create(:comment, project: project).user }
+    let!(:comment) { create :comment, commentable: project }
+    let!(:commenter) { create(:comment, commentable: project).user }
```

Comment:
Maybe I don't get it, but I don't see the point in creating two comment records here. Wouldn't this be enough?

```ruby
let!(:comment) { create :comment, commentable: project }
let(:commenter) { comment.user }